airbyte-webapp/src/components/connection/CatalogTree/next/StreamDetailsPanel/StreamPanelHeader/StreamPanelHeader.tsx

```diff
@@ -39,15 +40,25 @@ export const StreamPanelHeader: React.FC<StreamPanelHeaderProps> = ({
   onSelectedChange,
   stream,
 }) => {
+  const syncMode = (
+    <>
+      <FormattedMessage id={`syncMode.${config?.syncMode}`} />
+      {` | `}
+      <FormattedMessage id={`destinationSyncMode.${config?.destinationSyncMode}`} />
+    </>
+  );
   return (
     <div className={styles.container}>
       <div>
         <Switch small checked={config?.selected} onChange={onSelectedChange} disabled={disabled} />
       </div>
       <div className={styles.properties}>
-        <StreamProperty messageId="form.namespace" value={stream?.namespace} />
+        <StreamProperty
+          messageId="form.namespace"
+          value={stream?.namespace ?? <FormattedMessage id="form.noNamespace" />}
+        />
         <StreamProperty messageId="form.streamName" value={stream?.name} />
-        <StreamProperty messageId="form.syncMode" value={config?.syncMode} />
+        <StreamProperty messageId="form.syncMode" value={syncMode} />
       </div>
       <Button variant="clear" onClick={onClose} icon={<CrossIcon />} />
```

Comment:
The StreamProperty and the Close button do not match what's in figma:
![image](https://user-images.githubusercontent.com/168664/206004459-764358f9-7e22-41d7-a4dc-6d981d93a8c3.png)

Reply:
Fixed

<img width="1118" alt="image" src="https://user-images.githubusercontent.com/20929344/206203921-4076f192-5a3c-446d-98f8-0d05c78b9e2a.png">